src/table/utils/styling-utils.js

```diff
@@ -150,6 +150,21 @@ export function getBodyCellStyle(layout, theme) {
   };
 }
 
+export function getTotalsCellStyle(headerStyle, layout) {
+  if (getTotalPosition(layout) === 'noTotals') return false;
+  headerStyle.borderColor = 'rgb(166, 166, 166)';
+  headerStyle.fontWeight = 'bold';
```

Comment:
Not a great idea changing properties on an input argument, make sure you desctruct so you don't change the original `headerStyle`

Reply:
To not to get any conflict I created specific function for totalstyle